content/en/news/2019/announcing-1.3.1/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Announcing Istio 1.3.1
+description: Istio 1.3.1 release announcement.
+publishdate: 2019-10-27
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.1
+aliases:
+    - /about/notes/1.3.1
```

Comment:
No need for the aliases, aliases are only needed when moving an old page, and this is a new page.

Reply:
fixed. Thanks.

Reply:
Will the link be created automatically? This is the path that is referred by the release on GitHub.

Reply:
No, the old page will not be created. We need to update the GitHub entry to point to the new location for the page.

Reply:
OK, so we need to update the release script to do so.